src/rgw/rgw_op.cc

```diff
@@ -932,6 +934,22 @@ void RGWGetObj::execute()
   if (ret < 0)
     goto done_err;
 
+  keybl = attrs[RGW_ATTR_KEY];
+  ivbl = attrs[RGW_ATTR_IV];
+  mkeybl = attrs[RGW_ATTR_MKEY];
```

Comment:
do we get empty bufferlist for attributes which are not present ? or do we need to initialize these bufferlists also ?